### ISSUE TITLE
[1.7.1-rc7] Compiler plugins broken by final babel 7.0.0 update

### ISSUE THREAD

**Issue by erikolofsson** [meteor/meteor#10200]
[Update meteor-babel to version 7.0.0. 🎉](https://github.com/meteor/meteor/pull/9942/commits/dc8f4ffb13543042fee71e2624c82516f295e0b4) breaks https://github.com/sebakerckhof/meteor-minifiers-autoprefix

```
$ meteor create --release 1.7.1-rc.7 . --full
$ meteor add seba:minifiers-autoprefixer
Changes to your project's package version selections:
seba:minifiers-autoprefixer  added, version 1.1.0
seba:minifiers-autoprefixer: Standard minifiers used with Meteor apps by default + css autoprefixer.

$ meteor
[[[[[ /Source/testbabel2 ]]]]]

=> Started proxy.
=> Errors prevented startup:

   While loading plugin `minifyStdCSS` from package `seba:minifiers-autoprefixer`:
   packages/babel-runtime.js:51:13: Cannot read property '1' of null
   at babel-runtime.js (packages/babel-runtime.js:51:13)
   at fileEvaluate (packages/modules-runtime.js:322:7)
   at Module.require (packages/modules-runtime.js:224:14)
   at require (packages/modules-runtime.js:244:21)
   at packages/babel-runtime.js:101:15
   at packages/babel-runtime.js:108:3

=> Your application has errors. Waiting for file change.
```

Adding the meteor-minifiers-autoprefix package with METEOR_PACKAGE_DIRS so it builds locally works, so I guess that the problem is similar to https://github.com/meteor/meteor/pull/9942/commits/a52a2c28f148c6136ce5ddccade9d141fd1baf0e

Does this mean that all package compiler plugin packages needs to workaround this issue, and is it possible for them to support pre and post 1.7.1 meteor releases simultaneously?

**Comment by benjamn** [comment]
You're right, this is essentially the same problem that we fixed for the `coffeescript` package with https://github.com/meteor/meteor/pull/9942/commits/a52a2c28f148c6136ce5ddccade9d141fd1baf0e.

Eventually (hopefully in Meteor 1.7.2), we would like compiler plugins to be recompiled at application build time, but right now they are compiled when they are published, with whatever Meteor and npm package versions were current then, which in this case includes an older buggy version of the Meteor `babel-runtime` package.

@sebakerckhof You should be able to work around this problem by republishing `seba:minifiers-autoprefixer` with `babel-runtime@1.2.7` (the most recent published version). While you're at it, you might also want to include `"@babel/runtime": "7.0.0"` in your `npmDependencies` [here](https://github.com/sebakerckhof/meteor-minifiers-autoprefix/blob/b35e30624741917b2c317892fcbf282478b661ee/package.js#L14-L19), so that the plugin will have its own copy of `@babel/runtime` going forward, rather than relying on the one in Meteor's dev bundle. This will be safe in Meteor 1.7.0.5 and earlier, because those versions of Meteor don't have my commit https://github.com/meteor/meteor/pull/9942/commits/41b2f76ef3e9f7e52e1bd3477d58949f0e59d8d9, so the local `@babel/runtime` will be ignored in favor of the one in the dev bundle (buggy behavior that happens to be helpful here). I'm sorry for creating additional work for you, and of course I'm happy to answer any questions about this confusing situation.

**Comment by sebakerckhof** [comment]
Ok. I just released `seba:minifiers-autoprefixer@1.1.1` with the proposed changes and verified it with `1.7.1-rc.8`
